app/views/organisations/_contacts.html.erb

```diff
@@ -25,7 +25,7 @@
       <% if contact[:email_addresses].any? || contact[:links].any? || contact[:phone_numbers].any?  %>
         <div class="<%= contact[:post_addresses].any? ? "govuk-grid-column-one-half" : "govuk-grid-column-two-thirds" %>">
           <% if contact[:email_addresses].any? %>
-            <div class="organisation__margin-bottom">
+            <div class="organisation__margin-bottom organisation__paragraph">
```

Comment:
Feels a bit confusing to have a class name of `__paragraph` on a div element, could this be moved onto the `<p>` tag inside the element instead?

Reply:
Good point, will do 👍